README_SKINNY.rst

```diff
@@ -0,0 +1,14 @@
+=============================================
+MLflow Skinny: A Lightweight Machine Learning Lifecycle Platform Client
+=============================================
+
+MLflow Skinny is a lightweight MLflow package without SQL storage, server, UI, or data science dependencies.
+MLflow Skinny supports:
+* Tracking operations (logging / loading / searching params, metrics, tags + logging / loading artifacts
+* Model registration, search, artifact loading, and transitions
+* Execution of GitHub projects within notebook & against a remote target.
+
+Additional dependencies can be installed to leverage the full feature set of MLflow. For example:
+* To use the `mlflow.sklearn` component of MLflow Models, install `scikit-learn`, `numpy`, and `pandas`.
+* To use SQL-based metadata storage, install `sqlalchemy`, `alembic`, and `sqlparse`.
+
```

Comment:
For some reason, `twine` complains about the RST formatting when uploading to test PyPI. I think that the spacing of list items may be the culprit:

```suggestion
Additional dependencies can be installed to leverage the full feature set of MLflow. For example:

* To use the `mlflow.sklearn` component of MLflow Models, install `scikit-learn`, `numpy`, and `pandas`.
* To use SQL-based metadata storage, install `sqlalchemy`, `alembic`, and `sqlparse`.
```

Reply:
```
python -m readme_renderer README_SKINNY_PREFIX.rst -o README_SKINNY_PREFIX.html
<string>:1: (WARNING/2) Title overline too short.

=============================================
MLflow Skinny: A Light Weight Machine Learning Lifecycle Platform Client
=============================================
```

Reply:
Filed a PR to make it easier to detect issues like this one:

https://github.com/mlflow/mlflow/pull/4059/

Reply:
Thanks. that tool will be super helpful